components/aws/sagemaker/codebuild/integration-test.buildspec.yml

```diff
@@ -7,7 +7,7 @@ env:
 phases:
   pre_build:
     commands:
-      - docker login -u $DOCKER_CONFIG_USERNAME -p $DOCKER_CONFIG_PASSWORD
+      - aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/kubeflow-on-aws
```

Comment:
we should not be logging in to this ECR for integ and unit tests. Only in deploy
Can the unit and integ test jobs login to ECR in general, instead of this specific repo?

this docker login was added to get over ratelimiting because we had only one account before. Now that being said, lets update all the Dockerfiles in this repo to use the base images from ecr instead of dockerhub. I think all the images now have a replica in ecr


if we want to reduce the scope:
keep the docker login in unit and integ tests but for deploy, we will have to change the base image to be from ECR